Q/qwtw/build_tarballs.jl

```diff
@@ -11,6 +11,7 @@ sources = [
 ]
 
 # Bash recipe for building across all platforms
+apk add g++
```

Comment:
This needs to be in the script, at the beginning.

Reply:
@barche, maybe you can advice me about where to read about making a windows port from this script. I suspect this can be somewhere in BinaryBuilder help...

Reply:
I would start by setting `platforms = supported_platforms()` and then see where the errors appear.